include/vara/Feature/FeatureModel.h

```diff
@@ -96,8 +97,11 @@ class FeatureModel {
   string Name;
   fs::path Path;
   FeatureMapTy Features;
+  ConstraintContainerTy Constraints;
   Feature *Root;
 
+  FeatureModel() = default;
```

Comment:
Why do we add a default ctor for FeatureModels? Before we always wanted to have at least a path and a name.

Reply:
This way `FeatureModelBuilder` can inherit the data structure without already setting publicly required fields (like path / name) as they are build afterwards.